pkg/operator/configobservation/serviceca/observeserviceca.go

```diff
@@ -0,0 +1,76 @@
+package serviceca
+
+import (
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	"github.com/openshift/cluster-kube-controller-manager-operator/pkg/operator/configobservation"
+	"github.com/openshift/cluster-kube-controller-manager-operator/pkg/operator/operatorclient"
+	"github.com/openshift/library-go/pkg/operator/configobserver"
+	"github.com/openshift/library-go/pkg/operator/events"
+	"github.com/openshift/library-go/pkg/operator/resourcesynccontroller"
+)
+
+const (
+	serviceCAFilePath      = "/etc/kubernetes/static-pod-resources/configmaps/service-ca/service-ca.crt"
+	serviceCAConfigMapName = "service-ca"
+)
+
+// ObserveServiceCA fills in serviceServingCert.CertFile with the path for a configMap containing the service-ca.crt
+func ObserveServiceCA(genericListers configobserver.Listers, recorder events.Recorder, existingConfig map[string]interface{}) (map[string]interface{}, []error) {
+	listers := genericListers.(configobservation.Listers)
+	errs := []error{}
+	prevObservedConfig := map[string]interface{}{}
+
+	topLevelServiceCAFilePath := []string{"service-ca", "service-ca.crt"}
```

Comment:
`serviceServingCert.certFile` isn't it?

Reply:
Does it have to match exactly? I figured the file path would be independent of the config field.

Reply:
ignore me: 53e7e279